test/plug/adapters/translator_test.exs

```diff
@@ -1,6 +1,8 @@
 defmodule Plug.Adapters.TranslatorTest do
   use ExUnit.Case
 
+  @moduletag :cowboy1
```

Comment:
Let's move this test to the cowboy directory then.

Btw, doesn't cowboy2 also require changes to the translator stuff so we show errors properly?